### PR TITLE
MOTECH-2484 Changing language to 'tw.big5' breaks MOTECH - hide the C…

### PR DESCRIPTION
**Pull request by wstrzelczyk**
…hinese option

### CODE REVIEW

**Comment by wstrzelczyk** [comment]
Added hiding the Chinese option

File: platform/server-bundle/src/main/resources/webapp/js/controllers.js [hunk 1]
```diff
@@ -396,6 +396,7 @@
                 $scope.i18n = data;
             }),
             $scope.doAJAXHttpRequest('GET', 'lang/list', function (data) {
+                if(data['zh_TW.Big5'] !== undefined) { delete data['zh_TW.Big5']; }
```
Comment (sebbrudzinski) [inline platform/server-bundle/src/main/resources/webapp/js/controllers.js]: Let's style this code properly. The content of the if, as well as the closing brace should be in a new line. Moreover, let's add a comment with the ticket number, explaining that we temporarily hide this, until it's fixed.

File: platform/server-bundle/src/main/resources/webapp/js/controllers.js [hunk 2]
```diff
@@ -414,6 +415,7 @@
                 $scope.doAJAXHttpRequest('GET', 'lang/locate', function (data) {
                     $scope.i18n = data;
                 }), $scope.doAJAXHttpRequest('GET', 'lang/list', function (data) {
+                    if(data['zh_TW.Big5'] !== undefined) { delete data['zh_TW.Big5']; }
```
Comment (sebbrudzinski) [inline platform/server-bundle/src/main/resources/webapp/js/controllers.js]: Let's style this code properly. The content of the if, as well as the closing brace should be in a new line. Moreover, let's add a comment with the ticket number, explaining that we temporarily hide this, until it's fixed.

File: platform/server-bundle/src/main/resources/webapp/js/controllers.js [hunk 3]
```diff
@@ -528,6 +530,11 @@
 
         $scope.getStartupViewData = function() {
             $scope.doAJAXHttpRequest('GET', '../server/startupviewdata', function (data) {
+                if(data.languages['zh_TW.Big5'] !== undefined) { delete data.languages['zh_TW.Big5']; }
```
Comment (sebbrudzinski) [inline platform/server-bundle/src/main/resources/webapp/js/controllers.js]: Let's style this code properly. The content of the if, as well as the closing brace should be in a new line. Moreover, let's add a comment with the ticket number, explaining that we temporarily hide this, until it's fixed.

**Pull request merged**